plugin/aws/ec2Instance.go

```diff
@@ -64,16 +44,15 @@ const (
 	EC2InstanceStopped           = 80
 )
 
-func newEC2Instance(ctx context.Context, ID string, session *session.Session, client *ec2Client.EC2, attr plugin.Attributes) *ec2Instance {
+func newEC2Instance(ctx context.Context, ID string, session *session.Session, client *ec2Client.EC2) *ec2Instance {
 	ec2Instance := &ec2Instance{
 		EntryBase:        plugin.NewEntry(ID),
 		session:          session,
 		client:           client,
 		ssmClient:        ssm.New(session),
 		cloudwatchClient: cloudwatchlogs.New(session),
-		attr:             attr,
 	}
-	ec2Instance.TurnOffCachingFor(plugin.List)
+	ec2Instance.TurnOffCaching()
```

Comment:
I'd definitely like a comment here. `TurnOffCaching` doesn't really describe what's happening anymore either; it's more like `DisableDefaultCaching` (because it only affects the ops that are native to the plugin package).

Reply:
Agreed. I think `TurnOffCachingFor` => `DisableCachingFor`, `TurnOffCaching` => `DisableDefaultCaching` makes more sense.